### PR TITLE
set MAXPROCS according to real cpu quota

### PR DESCRIPTION
**Pull request by wangrzneu**
GOMAXPROCS needs to be set according to real CPU quota especially in container environment with CPU limit, but runtime.NumCPU() will return the CPU number of machine, which may be very large and will affect the performance. "go.uber.org/automaxprocs" can set the GOMAXPROCS according to the environment automatically.

### CODE REVIEW

**Review by gxthrj — DISMISSED** [review]

**Comment by EvanLjp** [comment]
let me fix the ci process, please wait for a moment

**Comment by gxthrj** [comment]
> let me fix the ci process, please wait for a moment

OK

File: go.mod [hunk 1]
```diff
@@ -14,6 +14,7 @@ require (
 	github.com/sirupsen/logrus v1.7.0
 	github.com/spf13/viper v1.7.1
 	github.com/urfave/cli/v2 v2.3.0
+	go.uber.org/automaxprocs v1.4.0
```
Comment (wu-sheng) [inline go.mod:17]: As this is a new runtime dependency, we need you to update the LICENSE.

**Review by wu-sheng — CHANGES_REQUESTED** [review]
Confirming LICENSE

File: go.mod (hunk 1 above)
Comment (wangrzneu) [reply go.mod:17]: Done

**Comment by wangrzneu** [comment]
@wu-sheng Please help to review it.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**